src/main/scala/Mathematics/FindMax.scala

```diff
@@ -0,0 +1,22 @@
+package Mathematics
+
+object FindMax {
+
+	/**
+	    * Method returns Max Element from the list
+	    *
+	    * @param listOfElements
+    	    * @return
+    	*/
+
+	def findMax(Elements : List[Int]): Int = {
+	    var length : Int = Elements.length;
```

Comment:
You can achieve the same thing without mutability:
`def findMax(elements : List[Int]): Int = elements.foldLeft(elements.head){(acc, i) => if (acc > i) acc else i}`

Reply:
done